src/lib/sw_engine/tvgSwStroke.cpp

```diff
@@ -844,6 +843,7 @@ void strokeReset(SwStroke* stroke, const RenderShape* rshape, const Matrix* tran
 
     stroke->width = HALF_STROKE(rshape->strokeWidth());
     stroke->cap = rshape->strokeCap();
+    stroke->miterlimit = rshape->strokeMiterlimit();
```

Comment:
It looks to be the proper location to convert the value, not in '_outside'.
stroke->miterlimit = stroke.miterlimit * (1 << 16);